cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -41,6 +41,25 @@ describe('Queue a transaction on 1/N', () => {
     // Insert max amount
     cy.contains('Max').click()
 
+    // Validates the "Max" button action, then clears and sets the actual sendValue
+    cy.get('input[name="tokenAddress"]')
+      .prev()
+      .find('p')
+      .contains(/G(ö|oe)rli Ether/)
+      .next()
+      .then((element) => {
+        const maxBalance = element.text().replace(' GOR', '').trim()
+        cy.wrap(element)
+          .parents('form')
+          .find('label')
+          .contains('Amount')
+          .next()
+          .find('input')
+          .should('have.value', maxBalance)
+          .clear()
+          .type(sendValue)
+      })
+
```

Comment:
I suggest to split this chunk in:
- Get max value to a variable
- Assert the input is correctly filled when clicking "Max"
- Clear the "Amount" input and type `sendValue`

Reply:
The problem here is that any variable set inside the `then()` will lose its value outside of that scope, even if you declare it outside of the `then()` like this:
```
let variable = 0
cy.get('something').then((element)=>{
variable = element.text()})
cy.get('somethingElse).should('cointain', variable)
```
That "variable" will be 0 no matter what you assigned inside the then, so the navigation and assertion must be done inside the brackets.
I could clear and type outside of the then, but i'd have to navigate to the input again for no reason